src/main/thrift/parquet.thrift

```diff
@@ -380,6 +410,38 @@ struct JsonType {
 struct BsonType {
 }
 
+/** Physical type and encoding for the geometry type */
+enum GeometryEncoding {
+  /**
+   * Allowed for physical type: BYTE_ARRAY.
+   *
+   * Well-known binary (WKB) representations of geometries.
+   */
+  WKB = 0;
+}
+
+/** Interpretation for edges of elements of a GEOMETRY type */
+enum Edges {
+  PLANAR = 0;
+  SPHERICAL = 1;
+}
+
+/**
+ * GEOMETRY logical type annotation (added in 2.11.0)
+ *
+ * GeometryEncoding and Edges are required. CRS is optional.
+ *
+ * Once CRS is set, it MUST be a key to an entry in the `key_value_metadata`
+ * field of `FileMetaData`.
```

Comment:
Why is it required that the CRS is embedded in file metadata? Isn't it clear if the CRS is a well-known one like `OGC:CRS84`? It seems to me that this resolution should be out of scope. Parquet can encourage that the CRS is documented in file metadata, but other systems could store the definition in a different location. For example, Iceberg could store this in a table property instead of in each data file.

I would prefer to define this string property as a "Coordinate reference system identifier" and not specify how to exchange the PROJJSON or other format definition. I would also add a note that people are encouraged to store it in a location along with the file or table metadata.

Reply:
@rdblue Although the GeoParquet community would really appreciate the possibility of embedding a full CRS description somewhere in Parquet, we understand that compromises need to be made sometimes.

Like @paleolimbot said, will it be acceptable that if we remove any conventions or requirements around the `string crs` and only allow this single value in the column metadata?

This means, the writer can put whatever they want but they will need to communicate this to the reader via other channel.

Reply:
> The need for embedding a full CRS description somewhere that is programatically accessible by a Parquet implementation is to ensure a producer's intent can be faithfully transported by the consumer.

To achieve this, is it possible to reserve some crs values or at least some prefixes? For example, Iceberg may store `iceberg.xxx` to crs where `xxx` is an arbitrary crs identifier defined in its table metadata. Similarly, GeoParquet may set `geoparquet.xxx`  to crs and the key must exist in the Parquet file metadata and its associated value is the full CRS.

This still causes fragmentation but it looks better than a strong enforcement. WDYT? @rdblue @jiayuasu @paleolimbot

Reply:
maybe `iceberg.geo.XXX` in table properties and `parquet.geo.xxx` in parquet file metadata? Not sure if this is allowed though.

Reply:
> Why is it required that the CRS is embedded in file metadata?

Note that it is not actually _required_ to embed CRS in the file metadata. This is an optional field, and so as a producer of Parquet files with geospatial data, you are not required to fill it. For example Iceberg, assuming it would already be tracking the CRS elsewhere in Iceberg-specific metadata or manifest file, could just leave this field blank in the parquet files itself.

Of course that makes those files less interoperable (but my understanding is that parquet files contained in an Iceberg table generally are not meant to be read by another non-Iceberg aware tool?). 
But putting something like `iceberg.xxx` as crs value would also not be great for interoperability.